demo 2/src/main/java/com/example/demo/service/OtpService.java

```diff
@@ -0,0 +1,102 @@
+package com.example.demo.service;
+
+import com.example.demo.constants.ResultInfoConstants;
+import com.example.demo.entity.Otp;
+import com.example.demo.exceptions.AccountNotFoundException;
+import com.example.demo.exceptions.DuplicateKeyException;
+import com.example.demo.repository.table.*;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Random;
+
+@RequiredArgsConstructor
+@Service
+@Slf4j
+public class OtpService {
+    private final AccountRepository accountRepository;
+
+    private final FolderRepository folderRepository;
+
+    private final ItemRepository itemRepository;
+
+    private final OtpRepository otpRepository;
+
+    public int generateOtp(Otp otp) {
```

Comment:
This method should just take in userId and then generate a otp and save in otp repository userId and otp. And return otp in response